modules/oe_webtools_etrans/oe_webtools_etrans.libraries.yml

```diff
@@ -0,0 +1,11 @@
+wt_etrans:
+  version: VERSION
```

Comment:
Why do we need this here?

Reply:
If I recall correctly, this allows the js to be rebuilt on drupal updates. I personally have a habit to put this.

Reply:
It is used as an identifier for your libraries. VERSION placeholder will just make \Drupal::VERSION, which is kind of pointless unless this is core module's library. Also, we had an issue with this placeholder used in a contrib: https://www.drupal.org/project/maxlength/issues/3326716

Reply:
That's interesting because I see related issues complaining about exactly the opposite of what this version var is supposed to do: https://www.drupal.org/project/klaro/issues/3497609. I'll remove it, but there might be further digging to do to get to the bottom of that.